openmc_plotter/main_window.py

```diff
@@ -83,7 +83,7 @@ def loadGui(self, use_settings_pkl=True):
         # Dock
         self.dock = DomainDock(self.model, self.font_metric, self)
         self.dock.setObjectName("Domain Options Dock")
-        self.addDockWidget(QtCore.Qt.RightDockWidgetArea, self.dock)
+        self.addDockWidget(QtCore.Qt.LeftDockWidgetArea, self.dock)
```

Comment:
Odd. The domain dock has appeared on the left in the plotter for as long as I can recall.